<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connexion</class>
 <widget class="QDialog" name="connexion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>connexion</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Pbu_Quitter">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="Pbu_Recherche">
       <property name="text">
        <string>. . .</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTableView" name="tableView1_BDD"/>
     </item>
     <item>
      <widget class="QTableView" name="tableView2_BDD"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="Table">
       <property name="text">
        <string>Table : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Chx_Table"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Colonne :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Chx_Colonne"/>
     </item>
     <item>
      <widget class="QLabel" name="Variable">
       <property name="text">
        <string>Variable :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Chx_Variable"/>
     </item>
     <item>
      <widget class="QLabel" name="id">
       <property name="text">
        <string>NÂ°ligne : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Num_Ligne"/>
     </item>
     <item>
      <widget class="QPushButton" name="Validation">
       <property name="text">
        <string>Validation</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>Pbu_Recherche</tabstop>
  <tabstop>Pbu_Quitter</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Pbu_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>connexion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1187</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Validation</sender>
   <signal>clicked()</signal>
   <receiver>tableView2_BDD</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Validation</sender>
   <signal>clicked()</signal>
   <receiver>tableView1_BDD</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>795</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
